### PR TITLE
Wire format review

### PR DESCRIPTION
**Pull request by kiniry**

### CODE REVIEW

File: docs/25_developer.md [hunk 1]
```diff
@@ -173,6 +174,22 @@ level, complete with versioning information. Note that we prefer that
 this dependency list is automatically generated and kept up-to-date by
 the build system.*
 
+### Apache Spark
```
Comment (ranweiler) [inline docs/25_developer.md]: Wrong Spark. The one we're using is Apache-licensed, but is not an Apache project: http://sparkjava.com/

Comment (kiniry) [inline docs/25_developer.md]: Whoops.  Now fixed.

File: docs/25_developer.md [hunk 2]
```diff
@@ -173,6 +174,39 @@ level, complete with versioning information. Note that we prefer that
...
+2. Create a database called "corla", and grant all privileges on it to
+   a user called "corla" with password "corla".
+   
+For example, to accomplish the above on OS X using brew, one issues
+the following commands:
+```brew install postgres
+   createuser -P corla
+   createdb -O corla corla```
+   
+That's it. If the database is there the server will use it (and will,
```
Comment (dmzimmerman) [inline docs/25_developer.md]: Note that there is no in-memory persistence anymore.

File: docs/25_developer.md [hunk 3]
```diff
@@ -173,6 +174,39 @@ level, complete with versioning information. Note that we prefer that
...
+at this stage, create all its tables and such automatically); if not,
+it will use in-memory persistence.
+
+*TBD: Describe the [Hibernate ORM](http://hibernate.org/orm/) and its
+use.*
+
+### Communication Formats
+
+*TBD: Describe
+the [Google GSON library and tools](https://github.com/google/gson).*
```
Comment (dmzimmerman) [inline docs/25_developer.md]: s/GSON/Gson

File: docs/25_developer.md (hunk 2 above)
Comment (kiniry) [inline docs/25_developer.md]: Fixed.

File: docs/25_developer.md (hunk 3 above)
Comment (kiniry) [inline docs/25_developer.md]: Ok, Mr. Pedantic. ;)

**Pull request merged**